controllers/messages.go

```diff
@@ -0,0 +1,41 @@
+package controllers
+
+import (
+	"github.com/perthgophers/puddle/models"
+	"html/template"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"path"
+)
+
+// locates the log and passes it
+func loadFile() (*models.Page, error) {
+	filename := path.Join("logs", "log.txt")
+	body, err := ioutil.ReadFile(filename)
+	if err != nil {
+		return nil, err
+	}
+	return &models.Page{Body: body}, nil
+}
+
+// locats and renders the html file
```

Comment:
spelling (locates)

also start this with `// renderTemplate...`

Also private funcs don't need to be commented. I like them though.